src/Flunt.Br/Document/CredCard.cs

```diff
@@ -0,0 +1,45 @@
+using Flunt.Br.Document.interfaces;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Flunt.Br.Document
+{
+    internal class CredCard : IValidate
```

Comment:
@marcoslcosta se essa validação for para cartão de crédito sugiro trocar para CreditCard assim como o método, trocar para IsCreditCard